src/components/Icon/icons/IconChevronRight.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC, ReactElement } from "react";
+
+import { SvgIconProps } from "../Icon";
+
+const IconChevronRight: FC<SvgIconProps> = ({
+  className = "",
```

Comment:
`className` isn't used - for other icons, `className={className}` is added to the `svg` elements, so I did this to mine for consistency!